### DISCUSSION TITLE
Feature request: add actions on table row select

### DISCUSSION THREAD

**Discussion by cfepdan** [Budibase/budibase#5296] [Ideas]
Navigating the page to a link when a table row is clicked is good... but I'd rather just change my form step instead. What if we had an onclick action instead or in addition?

**Comment by cfepdan** [comment]
Related: I could implement this in a repeater, if only I could make the repeater scrollable...

**Reply by melohagan** [reply]
Another thing too is that you can make a repeater scrollable fairly easily :)

**Build**
1. Setup your design tree:
![Screenshot 2022-04-07 at 15 36 06](https://user-images.githubusercontent.com/101575380/162224328-f2a81e41-c13a-45fd-9d8a-f0628a506984.png)

2. Set your container height to a fixed height. E.g. `400px`
3. Click *Edit custom CSS* and then enter the one-liner: `overflow: auto;`

**Result**
![Screenshot 2022-04-07 at 15 41 30](https://user-images.githubusercontent.com/101575380/162225530-4121b608-bbab-432d-9cc3-f537add181d7.png)

Happy building!

**Comment by melohagan** [comment]
One thing that can be done is to add a button to each row of your table, as discussed here: https://github.com/Budibase/budibase/discussions/5231

**Reply by cfepdan** [reply]
Brilliant!
